### PR TITLE
Add quotes for installation intsructions with optional dependencies

### PR DESCRIPTION
**Pull request by Bibo-Joshi**
We get questions like #3779 every now and then and adding the quotes doesn't hurt on systems that don't strictly need them afaik.

### CODE REVIEW

File: README_RAW.rst [hunk 1]
```diff
@@ -151,7 +151,7 @@ PTB can be installed with optional dependencies:
 
 * ``pip install python-telegram-bot-raw[passport]`` installs the `cryptography>=39.0.1 <https://cryptography.io/en/stable>`_ library. Use this, if you want to use Telegram Passport related functionality.
 * ``pip install python-telegram-bot-raw[socks]`` installs `httpx[socks] <https://www.python-httpx.org/#dependencies>`_. Use this, if you want to work behind a Socks5 server.
```
Comment (harshil21) [inline README_RAW.rst:153]:
```suggestion
* ``pip install "python-telegram-bot-raw[passport]"`` installs the `cryptography>=39.0.1 <https://cryptography.io/en/stable>`_ library. Use this, if you want to use Telegram Passport related functionality.
* ``pip install "python-telegram-bot-raw[socks]"`` installs `httpx[socks] <https://www.python-httpx.org/#dependencies>`_. Use this, if you want to work behind a Socks5 server.
```

File: README_RAW.rst [hunk 2]
```diff
@@ -151,7 +151,7 @@ PTB can be installed with optional dependencies:
...
-* ``pip install python-telegram-bot[http2]`` installs `httpx[http2] <https://www.python-httpx.org/#dependencies>`_. Use this, if you want to use HTTP/2.
+* ``pip install "python-telegram-bot[http2]"`` installs `httpx[http2] <https://www.python-httpx.org/#dependencies>`_. Use this, if you want to use HTTP/2.
 
 To install multiple optional dependencies, separate them by commas, e.g. ``pip install python-telegram-bot-raw[passport,socks]``.
```
Comment (harshil21) [inline README_RAW.rst:156]:
```suggestion
To install multiple optional dependencies, separate them by commas, e.g. ``pip install "python-telegram-bot-raw[passport,socks]"``.
```

File: README_RAW.rst (hunk 1 above)
Comment (Bibo-Joshi) [reply README_RAW.rst:153]: missed the `-raw` and the linebreak ones … Thanks for the catch!

**Review by harshil21 — APPROVED** [review]
lgtm

**Review by clot27 — APPROVED** [review]

**Pull request merged**